src/nbl/video/vulkan/profiles/NablaCore.json

```diff
@@ -1,2017 +1,2216 @@
-{
-    "$schema": "https://schema.khronos.org/vulkan/profiles-0.8.0-250.json#",
-    "capabilities": {
-        "baseline": {
-            "extensions": {
-                "VK_EXT_descriptor_indexing": 2,
-                "VK_EXT_external_memory_host": 1,
```

Comment:
I need this back in the Core profile, please find out why it disappeared (its probably beacuse you took apple/moltenvk profiles into account)

Reply:
it's because of android profiles (both Pixel 6A and 7) are missing them